src/custom/cappasity-cf-access-list.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Checks whether user has paid plan and adds to Cloudflare list
+ * @param  {Object} metadata
+ * @return {Promise}
+ */
+module.exports = async function addIpToAccessList({ user }, ctx) {
+  const { amqp, config: { router: { routes }, cfAccessList: { enabled } } } = this;
+  if (!enabled) return;
+
+  const { remoteip, audience } = ctx;
+  const route = `${routes.prefix ? `${routes.prefix}.` : ''}cf.add-to-access-list`;
+
+  if (remoteip !== false && user.metadata[audience].plan !== 'free') {
+    await amqp.publish(route, { remoteip });
```

Comment:
may want to add confirm, mandatory and deliveryMode flags for amqp, otherwise this may not be published

Reply:
fixed